frontend/OBSApp.cpp

```diff
@@ -15,108 +15,62 @@
     along with this program.  If not, see <http://www.gnu.org/licenses/>.
 ******************************************************************************/
 
-#include <time.h>
-#include <stdio.h>
-#include <wchar.h>
-#include <chrono>
-#include <ratio>
-#include <string>
-#include <sstream>
-#include <mutex>
-#include <filesystem>
-#include <util/bmem.h>
-#include <util/dstr.hpp>
-#include <util/platform.h>
-#include <util/profiler.hpp>
-#include <util/cf-parser.h>
-#include <obs-config.h>
-#include <obs.hpp>
-#include <qt-wrappers.hpp>
-#include <slider-ignorewheel.hpp>
-
-#include <QDir>
-#include <QFile>
-#include <QGuiApplication>
-#include <QScreen>
-#include <QProcess>
-#include <QAccessible>
-
-#include "obs-app.hpp"
-#include "obs-proxy-style.hpp"
-#include "log-viewer.hpp"
-#include "volume-control.hpp"
-#include "window-basic-main.hpp"
-#ifdef __APPLE__
-#include "window-permissions.hpp"
+#ifdef _WIN32
+#define WIN32_LEAN_AND_MEAN
```

Comment:
What was the rationale for moving `WIN32_LEAN_AND_MEAN` in the last force push?

Reply:
It's removed to the very top of all includes to ensure that any _other_ header that might include `windows.h` has that define set.

Reply:
We had previously preferred that the define is as close as possible to our own includes, perhaps in an effort to make it obvious when something requires a non-`WIN32_LEAN_AND_MEAN` `windows.h` include. I'm not sure if that changes things or not.